vhdbuilder/scripts/linux/mariner/tool_installs_mariner.sh

```diff
@@ -63,10 +79,10 @@ setMarinerNetworkdConfig() {
     DHCP=yes
     IPv6AcceptRA=no
 EOF
-# On Mariner 2.0 Marketplace images, the default systemd network config
+# On Mariner/AzureLinux Marketplace images, the default systemd network config
 # has an additional change that prevents Mariner from changing IP addresses
 # every reboot
-if [[ $OS_VERSION == "2.0" ]]; then 
+if [[ $OS_VERSION == "2.0" || $OS_VERSION == "3.0" ]]; then
```

Comment:
Is this still used for 1.0?

Reply:
I do not think any 1.0 images are being produced. Removed this check - [commit](https://github.com/Azure/AgentBaker/commit/25d763d2f861b8a5b2c7fca21aed83de0cc70715)